packages/widgets/src/stocks/component.tsx

```diff
@@ -0,0 +1,75 @@
+"use client";
+
+import { Sparkline } from "@mantine/charts";
+import { Flex, Stack, Text, Title, useMantineTheme } from "@mantine/core";
+import { IconTrendingDown, IconTrendingUp } from "@tabler/icons-react";
+
+import { clientApi } from "@homarr/api/client";
+import { useScopedI18n } from "@homarr/translation/client";
+
+import type { WidgetComponentProps } from "../definition";
+
+export default function StockPriceWidget({ options, width, height }: WidgetComponentProps<"stockPrice">) {
+  const t = useScopedI18n("widget.stockPrice");
+  const theme = useMantineTheme();
+  const [{ data }] = clientApi.widget.stockPrice.getPrices.useSuspenseQuery(options);
+
+  const stockData = data.chart.result[0];
+
+  const stockValues = Object.values(stockData.indicators.quote[0].close);
+  const stockValuesChange = (
+    100 *
+    ((stockValues[stockValues.length - 1] - stockValues[0]) / stockValues[stockValues.length - 1])
+  ).toFixed(2) as int;
```

Comment:
This is not true. It returns a string: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toFixed
Do not cast to ``int`` or ``number``